veevavault/1.3/modules/ROOT/pages/veevavault-connector-examples.adoc

```diff
@@ -163,176 +183,183 @@ image::veevavault-connector-get-documents-example.png[]
+			<on-error-propagate enableNotifications="true" logException="true" doc:name="On Error Propagate" type="ANY">
+				<logger level="INFO" doc:name="Logger" message="Flow Exception Handled --  #[payload]"/>
 			</on-error-propagate>
 		</error-handler>
 	</flow>
 ----
 
-== Validating a Spark Message
+[[validate-a-spark-message]]
+== Validate a Spark Message
 
-In this Mule flow, a Spark message triggered from the Vault is validated.
+This example triggers a Spark message from the Vault and validates the Spark message.
 
-To create the Mule flow:
+The following screenshot shows the Studio app flow for this example:
 
-. Drag and drop *HTTP Listener* from the Mule Palette to the project canvas.
-. The JKS Certificate for HTTPS Connector requires the .jks file for HTTPS Listener to receive Spark messages over a TLS connection with Veeva Vault. +
-Following are the steps to create a .jks file (you can use the Java Keytool located under the `<JDK_HOME>/bin` directory. The following is an example:
-	a.	Run the following command from the `<JDK_HOME>/bin` directory:
+image:veevavault-connector-spark-validator-example.png[Spark Validator operation Mule flow]
 
+. From the *Mule Palette* view, select *HTTP* and drag the *Listener* source to the canvas.
+. The JKS certificate for HTTP Connector requires the .jks file for HTTP Listener to receive Spark messages over a TLS connection with Veeva Vault. +
+The following steps show how to create a .jks file using the Java Keytool located under the `<JDK_HOME>/bin` directory:
+.	Run the following command from the `<JDK_HOME>/bin` directory:
++
 		`keytool -genkey -keyalg RSA -alias mule -keystore spark-listener.jks -storepass password -validity 360 -keysize 2048`
-
++
 		This creates a `spark-listener.jks` file in the current directory.
 
-	b.  Copy and import the `spark-listener.jks` file under the `src/main/resources` 	project directory.
-* Configure HTTPS Listener as follows:
-
-	a.	In the *General* tab, configure the following:
-
+. Copy and import the `spark-listener.jks` file under the `src/main/resources` 	project directory.
+. Configure HTTP Listener as follows:
+.. In the *General* tab, configure the following:
 		* Protocol: HTTPS
 		* Host: 0.0.0.0
 		* Port: 8082
 
-	b.	In *TLS Tab > Key Store*, configure the following:
-
+.. In the *TLS* tab, configure the following:
 		* Type: JKS
 		* Path: spark-listener.jks
 		* Alias: mule
 		* Key Password: password
 		* Password: password
 
-. Drag the *Spark Validator* operation from the Veeva Vault palette onto the project canvas.
-. Enter the values for the Veeva Vault connection: +
+. From the *Mule Palette* view, select *Veeva Vault* and drag the *Spark Validator* operation.
+. Enter the following values for the Veeva Vault connection: +
 * Username
 * Password
 * Vault URL
 * Vault version
 * Vault client ID
-. Configure the Spark Validator parameters as follows:
-
+. Configure the *Spark Validator* parameters as follows:
```

Comment:
Should these be field names instead of parameters (I think so)? If so, they should be bold.